test/service/ratelimit_test.go

```diff
@@ -139,8 +143,8 @@ func TestService(test *testing.T) {
 	// Config load failure.
 	t.configLoader.EXPECT().Load(
 		[]config.RateLimitConfigToLoad{{"config.basic_config", "fake_yaml"}}, gomock.Any()).Do(
-		func([]config.RateLimitConfigToLoad, stats.Scope) {
-			barrier.signal()
+		func([]config.RateLimitConfigToLoad, stats.Manager) {
+			defer barrier.signal()
```

Comment:
@mattklein123 I believe the CI build failed because of a race-condition in this test. The scenario is similar to the one fixed in [this recently merged PR](https://github.com/envoyproxy/ratelimit/pull/258).
I believe the issue should be resolved with this change.